rfcs/InputUnion.md

```diff
@@ -171,109 +183,144 @@ This section sketches out the potential goals that a solution might attempt to f
 
 Each criteria is identified with a Letter so they can be referenced in the rest of the document. New criteria must be added to the end of the list.
 
-### A) GraphQL should contain a polymorphic Input type
+### 🎯 A) GraphQL should contain a polymorphic Input type
 
 The premise of this RFC - GraphQL should contain a polymorphic Input type.
 
-### B) Input polymorphism matches output polymorphism
+| [1](#-1-explicit-__typename-discriminator-field) | [2](#-2-explicit-configurable-discriminator-field) | [3](#-3-order-based-discrimination) | [4](#-4-structural-uniqueness) | [5](#-5-one-of-tagged-union) |
+|----|----|----|----|----|
+| ✅ | ✅ | ✅ | ✅ | ⚠️ |
```

Comment:
What do those symbols mean exactly? A legend would be helpful.

Reply:
Added